src/collections/_documentation/workflow/integrations/integration-platform/index.md

```diff
@@ -3,166 +3,573 @@ title: Integration Platform
+        
+        payload = {
+            'user': 'pushover-user-key',
+            'token': 'pushover-api-token',
+            'message': event['message'][:1024],
+            'title': event['message'][:250],
+            'url': event['url'],
+            'url_title': 'Issue Details',
+            'priority': 0,
+        }
+        requests.post('https://api.pushover.net/1/messages.json', data=payload)
+
+    def get_sentry_issue(install_id, issue_id):
+        access_token = retrieve_from_db(install_id)
+        
+        url = 'https://sentry.io/api/0/issues/{}/'.format(issue_id)
+        headers = {'Authorization': 'Bearer {}'.format(access_token)}
+        
+        resp = requests.get(url, headers=headers)
+        return resp.json()
+```
+
+For more information, see the [full documentation on Webhooks](#webhooks).
+
+### Public Alerts
+
+There is an option called Alert Rule Action for the integration platform. What this means is that, when enabled, your integration will show up as a service in the action section when creating a new alert rule. For more information, see the [full documentation on Alert Rules]({%- link _documentation/workflow/notifications/alerts.md -%}).
+
+[SCREENSHOT: screenshot of the dropdown menu for alert rules ]
+
+For your service to receive webhooks for alert rules, users must add to existing rules or create new ones that have `Send a notification via <your service>` as an action in the rule. Once that's set up, you'll start receiving webhook requests for triggered alerts. For more information about the request and payload, see the [full documentation on Webhooks](#webhooks).
+
+### Published State
+
+When you are ready for the publication process, you can click the 'publish' button next to the integration you wish to submit. This will send an email to partners@sentry.io letting us know your integration is ready for review. 
+
+[ SCREENSHOT: screenshot of publish button ]
+
+## Internal Integrations
+
+Internal integrations are meant for custom integrations unique to your organization. They can also be as simple as an organization-wide token. Whether you are using just the API or all the Integration Platform features combined, internal integrations are for use within a single Sentry organization. 
+
+### Installation
+
+Creating an internal integration will automatically install it on your organization. 
+
+### Alerts
+
+Alerts are the same as public integrations -- see [Public Alerts](#public-alerts) for general information and see [Webhook Alerts](#webhook-alerts) for more detail on the request and payload. 
+
+### Integration Webhooks
+
+Since internal integrations are automatically installed (and uninstallation is essentially deleting the whole integration), there are no [un]installation webhooks. For more information, see the [full documentation on Webhooks](#webhooks).
+
+## API Token(s)
+
+Sentry's Integration Platform uses API Tokens, which are a similar concept to Access Tokens.
```

Comment:
I think we use the term 'Access Tokens' in the internal integrations blog post so we should probs either change that in the post or change it here.

Reply:
@mnoble What was the final decision regarding Access Tokens vs API Tokens?

Reply:
Let's change to Auth Tokens